src/common/components/ui/formik/AddButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, ButtonProps, ButtonVariant } from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { FieldArrayRenderProps } from 'formik';
+import React from 'react';
+
+interface AddButtonProps {
+  add: FieldArrayRenderProps['push'];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  addValue: any;
```

Comment:
better API for this component would be
```suggestion
  onAdd: VoidFunction
```

usage
```
<AddButton onAdd={() => push({ url: '', thumbprint: '' })} isInline>
  Add another Tang server
</AddButton>
```